CoronaContact/Logic/RiskCalculation/RiskCalculationController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  RiskCalculationController.swift
+//  CoronaContact
+//
+
+import ExposureNotification
+import Foundation
+import Resolver
+
+enum RiskCalculationError: Error {
+    case exposureDetectionFailed(Error)
+    case exposureInfoUnavailable(Error)
+    case cancelled
+}
+
+final class RiskCalculationController {
+    @Injected private var exposureManager: ExposureManager
+
+    private lazy var queue: OperationQueue = {
+        let queue = OperationQueue()
+        queue.maxConcurrentOperationCount = 1
+        return queue
+    }()
+
+    private let log = ContextLogger(context: .riskCalculation)
```

Comment:
For some reason I get a compiler error here that can be solved by being more explicit:
```suggestion
    private let log = ContextLogger(context: LoggingContext.riskCalculation)
```

Reply:
Pretty strange that I didn't receive the compiler error. I added the prefix just to be sure.